scripts/generate-crd-kustomize.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+REPO_ROOT=$(dirname ${BASH_SOURCE})/..
+
+KCONF_PACKAGE="github.com/kong/kubernetes-configuration"
+RAW_VERSION=$(go list -m -f '{{ .Version }}' ${KCONF_PACKAGE})
+if [[ $(echo "${RAW_VERSION}" | tr -cd '-' | wc -c) -ge 2 ]]; then
+    # If there are 2 or more hyphens, extract the part after the last hyphen
```

Comment:
Just to make it clear to the reader.
```suggestion
    # If there are 2 or more hyphens, extract the part after the last hyphen as 
    # that's a git commit hash (e.g. `v1.1.1-0.20250217181409-44e5ddce290d`).
```